query/src/configs/config.rs

```diff
@@ -238,15 +223,15 @@ impl fmt::Debug for MetaConfig {
     Clone, Debug, serde::Serialize, serde::Deserialize, PartialEq, StructOpt, StructOptToml,
 )]
 pub struct QueryConfig {
-    #[structopt(long, env = QUERY_TENANT, default_value = "", help = "Tenant id for get the information from the MetaStore")]
+    #[structopt(long, env = QUERY_TENANT, default_value, help = "Tenant id for get the information from the MetaStore")]
     #[serde(default)]
     pub tenant: String,
 
-    #[structopt(long, env = QUERY_NAMESPACE, default_value = "", help = "Namespace for construct the cluster")]
+    #[structopt(long, env = QUERY_NAMESPACE, default_value, help = "Namespace for construct the cluster")]
     #[serde(default)]
     pub namespace: String,
 
-    #[structopt(long, env = QUERY_NUM_CPUS, default_value = "0")]
```

Comment:
Ditto. `"0"` or `"8"`?

Reply:
Here we can keep it as `default_value = "0"`
But we should set it to `8` in tests/context to make the tests passed(such as explain).